crates/rover-client/Cargo.toml

```diff
@@ -10,6 +10,7 @@ publish = false
 [dependencies]
 apollo-federation-types = { workspace = true }
 chrono = { workspace = true, features = ["serde"] }
+console = { workspace = true }
```

Comment:
would be great to add the style to `rover_std` directly and only import `rover_std` here  - we try to keep all of our styles co-located there so that it's handled in the same way (mostly for `NO_COLOR` support but also if we ever wanted to change the underlying library we only have to do so in one place)

Reply:
Removes this import in https://github.com/apollographql/rover/pull/1637/commits/ed8e862789dfeb51f8ccc9a3c26a7b3432807aec (along with a few other changes).